models/models.js

```diff
@@ -14,5 +14,13 @@ const selectArticles = () => {
       return articles;
     });
 };
+const selectArticleById = (id) => {
+  let idArr = Object.values(id);
+  return db
+    .query("SELECT * FROM articles WHERE articles.article_id = $1;", idArr)
```

Comment:
I'd suggest just putting `[id]` in the query rather than saving the id in the new variable `idArr`, just a bit more readable than using Object.values, and it means you can remove line 18.